course/src/main/java/org/openedx/course/presentation/calendarsync/CourseContainerCalendarSyncUIState.kt

```diff
@@ -0,0 +1,14 @@
+package org.openedx.course.presentation.calendarsync
+
+import org.openedx.core.domain.model.CourseDateBlock
+import java.util.concurrent.atomic.AtomicReference
+
+data class CourseContainerCalendarSyncUIState(
```

Comment:
We can rename it to `CalendarSyncUIState` and combine both classes
CourseContainerCalendarSyncUIState + CourseDatesCalendarSyncUIState